packages/botonic-plugin-contentful/src/cms/factories/content-factories.ts

```diff
@@ -219,3 +222,37 @@ export class DocumentBuilder extends MessageContentBuilder {
     return new Document(this.buildCommonFields(), this.docUrl)
   }
 }
+
+export class HandoffBuilder extends MessageContentBuilder {
+  onFinish?: OnFinish
+  destination?: HandoffDestination
+  shadowing?: boolean
+  constructor(id: string, name: string, public text: string) {
+    super(id, name)
+  }
+
+  withOnFinish(onFinish?: OnFinish): this {
```

Comment:
if we don't allow OnFinish to be undefined
* remove "?"
* add argument to constructor